### PR TITLE
fixes to automatically use doc_type in the viewsets

### PR DESCRIPTION
**Pull request by rrmerugu**
this will automatically filter the doc types in the viewsets when multiple doc_type exist in the index.

### CODE REVIEW

File: src/django_elasticsearch_dsl_drf/viewsets.py [hunk 1]
```diff
@@ -39,7 +40,9 @@ def __init__(self, *args, **kwargs):
         self.client = connections.get_connection()
         self.index = self.document._doc_type.index
         self.mapping = self.document._doc_type.mapping.properties.name
-        self.search = Search(using=self.client, index=self.index)
+        doc_type = self.document.__name__
+        doc_type = re.sub(r'(.)([A-Z])', r'\1_\2', doc_type).lower()
+        self.search = Search(using=self.client, index=self.index, doc_type=doc_type)
```
Comment (barseghyanartur) [inline src/django_elasticsearch_dsl_drf/viewsets.py]: The `doc_type` value might be taken directly from `self.document._doc_type.name`. No need for fuzzy `re` guesses.
So, I think only the following line might stay:

```python
self.search = Search(
    using=self.client, 
    index=self.index, 
    doc_type=self.document._doc_type.name
)
```

Please, make changes and update the pull request.

Comment (rrmerugu) [reply src/django_elasticsearch_dsl_drf/viewsets.py]: sounds cool i will update that.. let me check

Comment (rrm-relishly) [reply src/django_elasticsearch_dsl_drf/viewsets.py]: yeah. made the change, works like a charm

File: src/django_elasticsearch_dsl_drf/viewsets.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 from .pagination import PageNumberPagination
 from .utils import DictionaryProxy
-
+import re
```
Comment (barseghyanartur) [inline src/django_elasticsearch_dsl_drf/viewsets.py]: This import becomes irrelevant and shall be removed.

File: src/django_elasticsearch_dsl_drf/viewsets.py (hunk 1 above)
Comment (barseghyanartur) [reply src/django_elasticsearch_dsl_drf/viewsets.py]: @rrmerugu:

Good. Thanks.

File: src/django_elasticsearch_dsl_drf/viewsets.py (hunk 2 above)
Comment (rrmerugu) [reply src/django_elasticsearch_dsl_drf/viewsets.py]: silly me !! will clean that

**Comment by barseghyanartur** [comment]
@rrmerugu:

Thanks.

**Comment by barseghyanartur** [comment]
@rrmerugu:

Released in version 0.6.4.

**Comment by rrmerugu** [comment]
@barseghyanartur  Great!

**Pull request merged**